source/includes/steps-configure-settings.yaml

```diff
@@ -0,0 +1,32 @@
+title: Open the Settings View.
+level: 4
+ref: open-settings
+content: |
+    .. list-table::
+       :header-rows: 1
+   
+       * - Operating System
+         - Method
+         - Actions
+
+       * - Any
+         - Visual Studio Code Menu
+         - :guilabel:`File` / :guilabel:`Code` > 
+           :guilabel:`Preferences` > 
+           :guilabel:`Settings`
+
+       * - MacOS
+         - Keyboard Shortcut
+         - Press ``Command`` + ``,``
```

Comment:
I worry users might think they need to press the plus sign on their keyboards. I'm not really sure the best way to fix this, but if you think of a way to clarify that we want Command and comma, feel free to update.